test/test_reactivities_bosch_hale.py

```diff
@@ -0,0 +1,66 @@
+import numpy.testing as npt
+import pytest
+
+## Testing DTn fusion reaction
+## Tested according to table VIII in H.-S. Bosch and G.M. Hale 1992 Nucl. Fusion 32 611
```

Comment:
This should go in the docstrings of the testing function

Reply:
Fixed with https://github.com/tmarkj/fusion-toolbox/commit/11b38930dc50d00fa48acc792eda03745692c0a4